index.html

```diff
@@ -1642,6 +1648,43 @@ <h3>
           </li>
         </ol>
       </section>
+      <section>
+        <h3>
+          <dfn>forget()</dfn> method
+        </h3>
+        <div class="note">
+          <p>
+            The user agent MAY decide to combine permissions across APIs, for
+            instance tracking WebHID + WebUSB device access under a unified
+            low-level device access permission. This is why this method may
+            also revoke additional (unspecified yet) permissions in the future.
+          </p>
+        </div>
+        <p>
+          The {{HIDDevice/forget()}} method steps are:
+        </p>
+        <ol>
+          <li>Let |promise:Promise| be [=a new promise=].
+          </li>
+          <li>Set [=this=].{{HIDDevice/[[state]]}} to `"forgetting"`.
+          </li>
+          <li>Perform the following steps [=in parallel=]:
+            <ol>
+              <li>Invoke the user agent to revoke access to all the [=HID interfaces=]
```

Comment:
I assume HIDDevice instances for other HID interfaces would also eventually become "forgotten" but currently there's no mechanism for this to happen. Let's do that here by iterating over all of {{HID/[[devices]]}} and repeating the "forgetting" steps for each HIDDevice that shares device access permissions with the original device.

Reply:
Done. Thanks!